### PR TITLE
feat(core): add globalcontext info to mixpanel track

### PR DESCRIPTION
**Pull request by EYHN**

### CODE REVIEW

**Comment by EYHN** [comment]
* **#7681** <a href="https://app.graphite.dev/github/pr/toeverything/AFFiNE/7681?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `canary`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @EYHN and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Review by JimmFly — APPROVED** [review]

File: packages/frontend/core/src/modules/telemetry/services/telemetry.ts [hunk 1]
```diff
@@ -25,14 +18,19 @@ import { WorkbenchService } from '../../workbench';
...
+    private readonly auth: AuthService,
+    private readonly globalContextService: GlobalContextService
+  ) {
     super();
   }
 
   onApplicationStart() {
     const account = this.auth.session.account$.value;
     this.updateIdentity(account);
+    this.registerMiddlewares();
```
Comment (pengx17) [inline packages/frontend/core/src/modules/telemetry/services/telemetry.ts:33]: I tried to use `ApplicationStarted` event for syncing tabs, however it seems this requires the service to be instantiated before this event has been emitted.

File: packages/frontend/core/src/pages/workspace/all-page/all-page.tsx [hunk 2]
```diff
@@ -25,6 +34,19 @@ export const AllPage = () => {
...
+  const isActiveView = useIsActiveView();
+
+  useEffect(() => {
+    if (isActiveView) {
+      globalContext.isAllDocs.set(true);
+
+      return () => {
+        globalContext.isAllDocs.set(false);
+      };
+    }
```
Comment (pengx17) [inline packages/frontend/core/src/pages/workspace/all-page/all-page.tsx:46]: may abstract this pattern into a new hook

File: packages/frontend/core/src/modules/telemetry/services/telemetry.ts (hunk 1 above)
Comment (EYHN) [reply packages/frontend/core/src/modules/telemetry/services/telemetry.ts:33]: This is not the case as long as the service is in the root scope

Comment (pengx17) [reply packages/frontend/core/src/modules/telemetry/services/telemetry.ts:33]: I guess it the service must be eagarly created

File: packages/frontend/core/src/pages/workspace/all-page/all-page.tsx (hunk 2 above)
Comment (EYHN) [reply packages/frontend/core/src/pages/workspace/all-page/all-page.tsx:46]: I'll think about how to abstract

**Review by pengx17 — APPROVED** [review]

**Pull request merged**